posix.mak

```diff
@@ -40,15 +40,16 @@ DFLAGS += -w -de
 DUBFLAGS = --arch=$(subst 32,x86,$(subst 64,x86_64,$(MODEL)))
 
 TOOLS = \
-    $(ROOT)/rdmd \
-    $(ROOT)/ddemangle \
     $(ROOT)/catdoc \
+    $(ROOT)/checkwhitespace \
+    $(ROOT)/ddemangle \
     $(ROOT)/detab \
+    $(ROOT)/rdmd \
     $(ROOT)/tolf
 
 CURL_TOOLS = \
-    $(ROOT)/dget \
-    $(ROOT)/changed
+    $(ROOT)/changed \
+    $(ROOT)/dget
 
 DOC_TOOLS = \
     $(ROOT)/dman
```

Comment:
This isn't checked atm (it requires dlang.org to be checked out - will do so in another PR)